src/data/resources.js

```diff
@@ -1,34 +1,30 @@
-// Define the data needed for the resources cards
+import CampGear from "../assets/CampGear.png";
+import CabinScrapbook from "../assets/CabinScrapbook.png";
+import TheSummit from "../assets/TheSummit.png";
+
 const ResourceItems = [
   {
-    name: "HACKER WELCOME GUIDE",
-    link: "http://google.com",
-    imageURL:
-      "https://upload.wikimedia.org/wikipedia/commons/9/90/Blank_flag_large.PNG",
-  },
-  {
-    name: "DEMO GUIDE",
-    link: "http://google.com",
-    imageURL:
-      "https://upload.wikimedia.org/wikipedia/commons/9/90/Blank_flag_large.PNG",
-  },
-  {
-    name: "PROJECT SUBMISSION RULES",
-    link: "http://google.com",
-    imageURL:
-      "https://upload.wikimedia.org/wikipedia/commons/9/90/Blank_flag_large.PNG",
+    name: "Camp Gear",
```

Comment:
could you add a "color" field here that tells you what color to render the card?